application/extractors.go

```diff
@@ -34,14 +31,6 @@ var URL Extractor = func(key string, req *Request) (interface{}, error) {
 		return nil, fmt.Errorf("URL %s is not valid", value)
 	}
 
-	mimetype := mime.TypeByExtension(filepath.Ext(value))
-
-	_, ok = image.Extensions[mimetype]
-
-	if !ok {
-		return nil, fmt.Errorf("Mimetype %s is not supported", mimetype)
-	}
```

Comment:
Great idea on this, but I suggest to think in a more generic way what if we let the developer choose its mimetype detector in the config?

By default we will be able to restore the default behavior with the extension (``MimetypeDetectorExtension``), we can add a ``NoopDetector`` to disable this and a ``FastImageDetector`` which is the enhanced version :))

Your opinion?